.gitignore

```diff
@@ -31,7 +31,6 @@ MANIFEST
 #  Usually these files are written by a python script from a template
 #  before PyInstaller builds the exe, so as to inject date/other infos into it.
 *.manifest
-*.spec
```

Comment:
Why?

Reply:
.spec file would give the ability to further configure PyInstaller.

Reply:
You don't have a spec file in your PR. Why remove it then?